### PR TITLE
Update mongodb dashboard with  enhancements including log panel

### PR DESCRIPTION
**Pull request by JonathanWamsley**
This pr depends on [ops agent mongodb enhancements](https://github.com/GoogleCloudPlatform/ops-agent/pull/777). This dashboard also includes an updated logs panel widget in place of the markdown.

### CODE REVIEW

File: dashboards/mongodb/metadata.yaml [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-    description: "This dashboard has several charts for the related [Google Cloud Ops Agent Integration for MongoDB](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb), including metrics like Active Connections, Collection Size, Server Memory Usage, Index Count, Index Size, and Operations."
+    id: overview
+    display_name: MongoDB Overview
+    description: |-
+      This dashboard is based on [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb#monitored-metrics) and has charts displaying Number of Connections, Number of Sessions, Number of Databases, Data Size, Storage Size, Number of Objects, Number of Collections, Number of Document Operations, Number of Indexes, Number of Indexes accessed, Index Size, Number of requests, Received Bytes, Transmitted Bytes, Operations Rate Avg., Operations Duration, Memory Usage, Cache Hits / Misses, Global Lock Hold Time, Number of Cursors, and Cursor Duration from MongoDB as well as charts of infrastructure related metrics for the running MongoDB VMs: CPU % Top 5 VMs, Memory % Top 5 VMs, and Hosts by Region.
```
Comment (antonblock) [inline dashboards/mongodb/metadata.yaml:7]: I think `Number of x` can be replaced with `x Count` here, or `Number of` could just be dropped

Comment (JonathanWamsley) [reply dashboards/mongodb/metadata.yaml]: updated!

**Comment by JonathanWamsley** [comment]
Hello @EvanSimpson  , the [mongodb enhancements](https://github.com/GoogleCloudPlatform/ops-agent/pull/777) has been added to the ops agent and this pr is ready for review! Thank you

File: dashboards/mongodb/metadata.yaml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 sample_dashboards:
   -
     category: MongoDB
-    id: mongodb-gce-overview
-    display_name: MongoDB GCE Overview
-    description: "This dashboard has several charts for the related [Google Cloud Ops Agent Integration for MongoDB](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb), including metrics like Active Connections, Collection Size, Server Memory Usage, Index Count, Index Size, and Operations."
+    id: overview
+    display_name: MongoDB Overview
+    description: |-
+      This dashboard is based on [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb#monitored-metrics) and has charts displaying Connections, Sessions, Databases, Data Size, Storage Size, Objects, Collections, Document Operations, Indexes, Indexes accessed, Index Size, Requests, Received Bytes, Transmitted Bytes, Operations Rate Avg., Operations Duration, Memory Usage, Cache Hits / Misses, Global Lock Hold Time, Cursors, and Cursor Duration from MongoDB as well as charts of infrastructure related metrics for the running MongoDB VMs: CPU % Top 5 VMs, Memory % Top 5 VMs, and Hosts by Region.
```
Comment (EvanSimpson) [inline dashboards/mongodb/metadata.yaml:7]:
```suggestion
      This dashboard is based on [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb#monitored-metrics) and has charts displaying Connections, Sessions, Databases, Data Size, Storage Size, Objects, Collections, Document Operations, Indexes, Indexes accessed, Index Size, Requests, Received Bytes, Transmitted Bytes, Operations Rate Avg., Operations Duration, Memory Usage, Cache Hits / Misses, Global Lock Hold Time, Cursors, and Cursor Duration from MongoDB as well as charts of infrastructure related metrics for the VMs running MongoDB: CPU % Top 5 VMs, Memory % Top 5 VMs, and Hosts by Region.
```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/mongodb/metadata.yaml [hunk 3]
```diff
@@ -1,9 +1,11 @@
 sample_dashboards:
   -
     category: MongoDB
-    id: mongodb-gce-overview
-    display_name: MongoDB GCE Overview
-    description: "This dashboard has several charts for the related [Google Cloud Ops Agent Integration for MongoDB](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb), including metrics like Active Connections, Collection Size, Server Memory Usage, Index Count, Index Size, and Operations."
+    id: overview
```
[the remark this answers is not in the document]
Comment (JonathanWamsley) [reply dashboards/mongodb/metadata.yaml]: updated

File: dashboards/mongodb/metadata.yaml (hunk 2 above)
Comment (JonathanWamsley) [reply dashboards/mongodb/metadata.yaml]: updated

**Comment by JonathanWamsley** [comment]
Hello @EvanSimpson , the changes you requested have been made!

**Review by EvanSimpson — APPROVED** [review]
Looks good! I'm going to wait to merge until we've updated the mongo integration minimum agent version to 2.19.0 from the upstream ops-agent repo.

**Pull request merged**